src/game/ticTacToeGame.ts

```diff
@@ -98,6 +98,20 @@ export class TicTacToeGame {
     return isGameOver(this.getBoardState());
   }
 
+  potentialMoveOptions = (square: Square[], board = this.getBoardState()) => {
```

Comment:
Let's call this `squares` instead of `square` to make it more obvious we're dealing with an array.